roles/openshift_logging/tasks/generate_es_pems.yaml

```diff
@@ -0,0 +1,37 @@
+---
+# is_client
+# add_oid
+# certs
+
+- name: Checking for {{ component }}.key
+  stat: path="{{ generated_certs_dir }}/{{ component }}.key"
+  register: key_file
+  check_mode: no
+
+- name: Checking for {{ component }}.crt
+  stat: path="{{ generated_certs_dir }}/{{ component }}.crt"
+  register: cert_file
+  check_mode: no
+
+- set_fact:
+    cert_ext: "{{ certs | strflist(pattern='DNS.{@}:{}', index_start=2, ignore_empty_key=True) }}"
+
+- set_fact:
+    cert_ext: "{{ cert_ext + ['RID.1:1.2.3.4.5.5'] }}"
```

Comment:
Do we know where this is configured in SG.  This PR[1] was the first attempt to use the OID specific to our organization

[1] https://github.com/openshift/origin-aggregated-logging/pull/319

Reply:
https://github.com/floragunncom/search-guard/blob/fd43977b779081555ab4d1b5cc775f5e2f34349a/src/main/java/com/floragunn/searchguard/support/ConfigConstants.java#L155

Reply:
Though, if we aren't going to auto gen our certs each time, i'd be a little hesitant to set that... yet.